app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -130,10 +130,7 @@ protected void onStart() {
 
         mGoogleApiClient.registerConnectionCallbacks(this);
         mGoogleApiClient.registerConnectionFailedListener(this);
-
-        if (PrefUtils.isSignedIn(this)) {
-            mGoogleApiClient.connect();
-        }
+        mGoogleApiClient.connect();
```

Comment:
We us the mGoogleApiClient to load chapter details e.g. in EventOverviewFragment. Hence, it feels wrong not to connect at all.